### PR TITLE
implemented withdraw

### PR DESCRIPTION
**Pull request by frosty00**

### CODE REVIEW

File: js/livecoin.js [hunk 1]
```diff
@@ -488,6 +489,22 @@ module.exports = class livecoin extends Exchange {
         throw new ExchangeError (this.id + ' cancelOrder() failed: ' + this.json (response));
     }
 
+    async withdraw (currency, amount, address, tag = undefined, params = {}) {
+        // Sometimes the response with be { key: null } for all keys.
+        // An example is if you attempt to withdraw more than is allowed when withdrawal fees are considered.
```
Comment (kroitor) [inline js/livecoin.js]: Thanks again for your participation! Really appreciated! Can you please elaborate on these two comment lines above? I mean, how does the actual response look like? I see `{ key: null }` there, but in L504 we are using the `id`, therefore I'm a little confused...

Comment (frosty00) [reply js/livecoin.js]: This is one that was succesful:

```
 { fault: null,
     userId: 103605,
     userName: 'frosty00',
     id: 511413616,
     state: 'APPROVED',
     createDate: 1520373827066,
     lastModifyDate: 1520373827947,
     comment: null,
     description: 'Transfer from Livecoin',
     amount: 0.01418088,
     currency: 'ETH',
     wallet: '0xd9259302c32c0a0295d86a39185c9e14f6ba0a0d' }
```

Whereas one that fails is sometimes 

```
   { fault: null,
     userId: null,
     userName: 'null',
     id: null,
     state: 'null',
     createDate: null,
     lastModifyDate: null,
     comment: null,
     description: null,
     amount: null,
     currency: 'null',
     wallet: 'null' },
```

But only two things I have found trigger this.

* The signiture is wrong (I was using amount.toString ()
* accountBalance > amount and accountBalance < amount + withdrawalFee

Comment (kroitor) [reply js/livecoin.js]: Got it, thanks for the explanation! The amount.toString will be reworked soon, as you know (as you did it ))), and I'll think of ways of working around the second issue with the account balance... probably will add some clarifying warning message or something, will see. Thx once again!

**Comment by kroitor** [comment]
Nice one ) 👍 

screen shot 2018-03-07 at 02 24 23

**Pull request merged**